src/main/java/uk/gov/hmcts/darts/Application.java

```diff
@@ -28,13 +28,7 @@ public static void main(final String[] args) {
 
     @EventListener(ApplicationReadyEvent.class)
     public void runAfterStartup() {
-        log.info(String.format(
-            "ATS ffmpegExecutable: %s, concatWorkspace: %s, mergeWorkspace: %s, trimWorkspace: %s",
-            audioTransformConfigurationProperties.getFfmpegExecutable(),
-            audioTransformConfigurationProperties.getConcatWorkspace(),
-            audioTransformConfigurationProperties.getMergeWorkspace(),
-            audioTransformConfigurationProperties.getTrimWorkspace()
-        ));
+        log.info(audioTransformConfigurationProperties.toString());
```

Comment:
It's just logging the audiotransform properties we have configured e.g. ```AudioTransformConfigurationProperties(ffmpegExecutable=ffmpeg, concatWorkspace=/Users/richardbull/audiotransform/concatenate, mergeWorkspace=/Users/richardbull/audiotransform/merge, trimWorkspace=/Users/richardbull/audiotransform/trim, reEncodeWorkspace=/Users/richardbull/audiotransform/encode)```

I'm happy to remove / update the properties as we want them - let me know

Reply:
Don't think we need to log the config details anywhere but it would be good to have a test which proves FFMPEG command done via tar download really works. e.g. ffmpeg -version. May be under another ticket?

Reply:
@hemantasharma1129 / @jackmaloney Replaced this with a method that will run the `ffmpeg -version` command after the Application is started. You would expect to see `ffmpeg version 6.0-static` in the linux container application log

Reply:
@hemantasharma1129 Removed `ffmpegShowVersionCommand` in commit cc393d0 because this will fail the integration tests in the build pipeline - https://github.com/hmcts/darts-api/actions/runs/4978850140/jobs/8909683299?pr=92